problems/loops.js

```diff
@@ -131,7 +199,23 @@ while (i > 3) {
  * @returns {Array} [1, 2, "Fizz", 4, "Buzz"... 98, "Fizz", "Buzz"]
 
  */
-function fizzBuzzPart1() {}
+function fizzBuzzPart1() {
+  let arr = []
+  let num = 1
+  while (num <= 100) {
+    if (num % 3 === 0) {
+      arr.push("Fizz")
+    }
+    else if (num % 5 === 0) {
```

Comment:
The number 15 will pass this else if statement but it shouldn't because it is also divisible by 3